src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -72,33 +82,25 @@ const OrgProfileTab: FC = () => {
         src={org.name}
         isRenameableOrg={true}
       />
-      {CLOUD && hasIdentityData && (
+      {CLOUD && fetchedOrgDetails && (
         <>
           <FlexBox
+            className="org-profile-tab--details"
             direction={FlexDirection.Row}
             margin={ComponentSize.Medium}
             justifyContent={JustifyContent.SpaceBetween}
             stretchToFitWidth={true}
-            style={orgProviderExists ? {width: '85%'} : {width: '48%'}}
             testID="org-profile--labeled-data"
           >
-            {orgProviderExists && (
-              <LabeledData label="Cloud Provider" src={quartzOrg.provider} />
-            )}
-            {quartzOrg.regionCode && (
-              <LabeledData label="Region" src={quartzOrg.regionCode} />
-            )}
-            {quartzOrg.regionName && (
-              <LabeledData label="Location" src={quartzOrg.regionName} />
-            )}
+            <LabeledData label="Cloud Provider" src={quartzOrg.provider} />
```

Comment:
All of these checks are extra verbiage. These properties are required, so if the API call comes back we don't need to do individualized guards.